src/main/java/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
```

Comment:
객체지향 생할 체조 원칙 중 `규칙 5: 줄여쓰지 않는다(축약 금지).`를 참고해주세요.
```suggestion
public class StringAddCalculator {
```

Reply:
수정완료